eng/pipelines/dotnet-buildtools-prereqs-all.yml

```diff
@@ -32,12 +41,19 @@ variables:
 - template: /eng/pipelines/variables/common.yml@self
   parameters: 
     disableMatrixTrimming: ${{ parameters.disableMatrixTrimming }}
+    sourceBuildPipelineRunId: ${{ parameters.sourceBuildPipelineRunId }}
 - name: publishEolAnnotations
   value: true
 
 extends:
   template: /eng/common/templates/1es-official.yml@self
   parameters:
+    serviceConnections:
+    - name: $(internal-mirror.serviceConnectionName)
+    - name: $(build.serviceConnectionName)
+    - name: $(publish.serviceConnectionName)
+    - name: $(kusto.serviceConnectionName)
```

Comment:
This pipeline doesn't ingest kusto data: https://dev.azure.com/dnceng/internal/_build/results?buildId=2741686&view=logs&j=739bb7ec-7296-5b40-7dff-ee3297bfdaa4&t=8527e9f7-1005-5ad0-5483-90cbe0361998